.circleci/config.yml

```diff
@@ -0,0 +1,47 @@
+version: 2.1
+
+executors:
+  default:
+    working_directory: ~/repo
+    docker:
+      - image: circleci/node:8.11.3
```

Comment:
I think latest or Dubnium is better

Reply:
I agree. I'll update that on the next update